### ISSUE TITLE
Propper credit and lots of bugfixes

### ISSUE THREAD

**Issue by AJenbo** [yousifd/devilution-vita#1]
Hi, would you be willing to rebase this on https://github.com/diasurgical/devilutionX so that proper credit is given to all that worked to make this possible. This would also fix a lot of the defects and issues with the port. Let us know if you need some help with the process.

**Comment by yousifd** [comment]
Yeah I could do that. I'm sorry if I didn't do this from the start, I was just hacking around and I guess I forgot to make this repo private. I'll make the changes as soon as I can.

**Comment by AJenbo** [comment]
No worries at the time you did this DevilutionX was still 3 months away from fully covering all the needs for this port. It would be really awesome if you can help to get Vita support upstreamed so that we can better support players that are interested and have Hellfire support ready on day one :)
